saleor/graphql/schema.graphql

```diff
@@ -6200,20 +6202,19 @@ type ShippingMethod implements Node & ObjectWithMetadata {
   id: ID!
   privateMetadata: [MetadataItem]!
   metadata: [MetadataItem]!
+  type: ShippingMethodTypeEnum @deprecated(reason: "\\n\\nDEPRECATED: this field will be removed in Saleor 4.0.")
```

Comment:
Are those new line chars on purpose?

Reply:
Good catch 👍  I used `DEPRECATED_IN_3X_INPUT` instead of `DEPRECATED_IN_3X_FIELD`